### PR TITLE
Add Columbus file

### PR DESCRIPTION
**Pull request by bjswick33**
This is the beginning of the Columbus file. :football: 

Please provide feedback on:
- if you have ever been to Columbus, OH, what did you like? what did you not like?
- if you have never been to Columbus, OH, what interests you about coming here?

### CODE REVIEW

**Comment by diarmaidm** [comment]
> ever been to Columbus, OH,

Not yet....

File: bjswick33-columbus.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Columbus, OH
```
Comment (KimThomp) [inline bjswick33-columbus.md]: I have lots of Fam in Columbus. :+1:

**Comment by bjswick33** [comment]
Cool!  I'm sure you would like it when you have time to visit.

**Comment by diarmaidm** [comment]
:+1: :shipit:

**Pull request merged**